crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -1026,6 +1026,8 @@ def finalize_node_upgrade(node)
       # to ensure that no old services remain running and cause restarted Nova
       # services to autonegotiate the old RPC API version.
       def stop_nova_services
+        # Only related to non-disruptive mode where nodes will be fully rebooted
```

Comment:
This comment is not very clear to me... aren't nodes rebooted independent of the upgrade mode?

Reply:
Oh, I did not read it after myself